layout/netstandard20/project.json

```diff
@@ -2,7 +2,7 @@
   "frameworks": {
     "netstandard1.7": {
       "dependencies": {
-        "NETStandard.Library2": "2.0.0-beta-24801-0"
+        "NetStandard.Library2": "2.0.0-beta-24801-0"
```

Comment:
You're right. It's my confusion.
We need to fix `src/System.Security.Cryptography.Xml/src/project.json` (NetStandard -> NETStandard), not this.